src/features/dashboard/settings/profile-form.tsx

```
@@ -73,44 +155,102 @@ export function ProfileForm() {
-                                </Avatar>
-                                <div className="absolute inset-0 bg-black/40 rounded-full flex items-center justify-center opacity-0 group-hover:opacity-100 transition-opacity">
-                                    <Camera className="h-6 w-6 text-white" />
+                    <Form {...form}>
+                        <form onSubmit={form.handleSubmit(onSubmit)} className="space-y-6">
+
+                            {/* Avatar Section */}
+                            <div className="flex items-center gap-4">
+                                <div
+                                    className="relative group cursor-pointer"
+                                    onClick={triggerFileInput}
+                                >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)